containers/iannotatesv/run_iannotatesv.py

```diff
@@ -1,51 +1,68 @@
 import argparse, sys, os, subprocess
+from collections import OrderedDict
+from datetime import datetime
+import multiprocessing as mp
 import numpy as np
 import pandas as pd
+from utils import *
 
 def usage():
 	parser = argparse.ArgumentParser()
 	parser.add_argument('--bedpe', help = 'bedpe file', required = True)
 	parser.add_argument('--genome', default = "hg19", help = 'hg19/hg38/hg18')
+	parser.add_argument('--threads', type=int, default = 4, help = 'number of threads for parallelization')
+	parser.add_argument('--chunk', type=int, dest = "chunk_size", default = 500, help = 'how many records to process with a single thread')
```

Comment:
Can `--chunk` be calculated as an optimized/reasonable value based on number of record in the bedpe file and number of threads was given? I guess it also related to memory as well. Maybe memory limit will also get involved here. So maybe a preset formula can be defined here to cover the most basic needs? 
I'm thinking this might be a bit difficult for user to figure out a best number here with limited knowledge.

Reply:
will remove chunk param.